src/common.cpp

```diff
@@ -294,8 +294,8 @@ std::list<Point_t> map_2_goals(std::vector<std::vector<bool> > const& grid, bool
 {
   std::list<Point_t> goals;
   int ix, iy;
-  uint nRows = grid.size();
-  uint nCols = grid[0].size();
+  int nRows = grid.size();
```

Comment:
`size_t`

Reply:
this change imply changes in  the Point_t and more parts of the code. Since this PR is aimed to focus on the conversion to ROS2 part, I suggest to do these changes in another PR